ladybug/sunpath.py

```diff
@@ -780,7 +780,7 @@ def _calculate_sun_vector(self):
             .rotate_around(z_axis, self.azimuth_in_radians) \
             .rotate_around(z_axis, math.radians(-1 * self.north_angle))
 
-        _sun_vector.normalize().flip()
+        _sun_vector.normalize()  #.flip()
```

Comment:
This is actually important and changes the direction of vector. I might have added it myself. If it results in incompatibility we can do something like:

```python
 _sun_vector.normalize()
_sun_vector =  Vector3(-1 * _sun_vector.x, -1 * _sun_vector.y, -1 * _sun_vector.z)
```